### PR TITLE
Removed NOTIFICATIONS_WIP_ENABLED flag

### LINKED ISSUE #2813: Flag hygiene: Remove NOTIFICATIONS_WIP_ENABLED flag
### User Story

As a developer I want flags to be up to date so we know what behaviors we want to toggle.

### Description

*Type*: Feature/Refactor

*Summary*: Remove NOTIFICATIONS_WIP_ENABLED flag while keeping behavior for flag ON.

See https://github.com/status-im/status-react/blob/develop/src/status_im/utils/config.cljs#L19

#### Expected behavior
PNs work with flag.

#### Actual behavior
Same behavior without flag.

### PR DESCRIPTION
**Pull request by taylorjdawson**
Fixes #2813

### Summary:
Removed NOTIFICATIONS_WIP_ENABLED flag while keeping behavior for flag ON.

status: ready

### CODE REVIEW

File: src/status_im/ios/core.cljs [hunk 1]
```diff
@@ -43,10 +43,9 @@
          (.hide react/splash-screen))
        :component-did-mount
        (fn []
-         (when config/notifications-wip-enabled?
            (notifications/request-permissions)
```
Comment (flexsurfer) [inline src/status_im/ios/core.cljs]: indentation

File: src/status_im/android/core.cljs [hunk 2]
```diff
@@ -68,10 +68,9 @@
          (.addEventListener react/app-state "change" app-state-change-handler))
        :component-did-mount
        (fn []
-         (when config/notifications-wip-enabled?
            (notifications/on-refresh-fcm-token)
```
Comment (flexsurfer) [inline src/status_im/android/core.cljs]: indentation

File: src/status_im/utils/notifications.cljs [hunk 3]
```diff
@@ -22,9 +22,8 @@
 (defn get-fcm-token []
     (-> (.getFCMToken (aget rn/react-native-fcm "default"))
         (.then (fn [x]
-                 (when config/notifications-wip-enabled?
                    (log/debug "get-fcm-token: " x)
```
Comment (flexsurfer) [inline src/status_im/utils/notifications.cljs]: indentation

**Review by flexsurfer — CHANGES_REQUESTED** [review]

**Comment by taylorjdawson** [comment]
Alrighty fixed the indentation, thanks!

**Review by flexsurfer — APPROVED** [review]

**Review by oskarth — APPROVED** [review]

**Comment by asemiankevich** [comment]
I have tested this with 2 builds. I built it with flag = 1 in Jenkins and flag = 0 . I am able to receive PNs in both cases so I assume there is no more impact from the mentioned flag. However the flag is still visible in Jenkins build configuration which is not in scope of this PR (as I got from @oskarth ) and this setting will be removed in one of future PRs. 


Branch: taylorjdawson:feature/remove-notifications-wip-enabled-#2813
Android: https://i.diawi.com/t1QD5d
iOS: https://i.diawi.com/s8vWvD

@taylorjdawson I believe you need to resolve some conflicts before the PR is merged

**Comment by andytudhope** [comment]
Yeah @taylorjdawson - please just rebase onto latest develop to solve these conflicts and then we will merge and award here 👍

**Comment by oskarth** [comment]
>  However the flag is still visible in Jenkins build configuration which is not in scope of this P

Reminder: Remove from Jenkins as this PR is merged (after conflicts solved)

**Pull request merged**